package.json

```diff
@@ -72,7 +72,7 @@
     "dalliance": "git+https://github.com/ENCODE-DCC/dalliance.git#v1.0",
     "domready": "^0.3.0",
     "form-serialize": "^0.6.0",
-    "genome-visualizer": "github:VALIS-software/valis-hpgv",
+    "genome-visualizer": "git+https://github.com/ENCODE-DCC/valis-hpgv.git",
```

Comment:
I had a question about this. It looked like the old line pointed to the original Valis repo, but somehow we use the one you forked? Ben said we use “ours.”

Reply:
we forked it and now we use that one (until this PR they were identical)

Surprised no one caught the stray dalliance dependency!  That should go.  (Eventually)

Reply:
I've removed it! Thank you for catching that.